testing/session.go

```diff
@@ -273,14 +273,10 @@ func ExpireSession(
 			err := s.ExpireSession(ctx, tt.args.key)
 			diffPlatformErrors(tt.name, err, tt.wants.err, opPrefix, t)
 
-			session, err := s.FindSession(ctx, tt.args.key)
+			_, err = s.FindSession(ctx, tt.args.key)
 			if err == nil {
 				t.Errorf("expected session to be expired got %v", err)
 			}
-
```

Comment:
Why are we dropping this comparison? I fixed it up recently because we missed a bug but not implementing it correctly. Which is why I ask.

Reply:
The test was requiring an error to be returned as well as a session object. I came across a todo about this [here](https://github.com/influxdata/influxdb/pull/17950/files#diff-0e9e292791dba4c98b67f966ab4b9081L82) and decided to bring the behavior of this function call in line with the way we usually write go (either object or error)

Reply:
Ah OK cool, I see. If this test expects an error and a nil session, then perhaps we should assert the session is nil and check the error is the specific expired error.

Reply:
I see as well I was wrong. It was the test above that I fixed up. My mistake.

Reply:
ill add a check here to make sure session is nil :)